### ISSUE TITLE
Ruby 2.3.3 / 2.4.0

### ISSUE THREAD

**Issue by hwhelchel** [netlify/build-image#45]
Could you please add ruby 2.3.3 and 2.4.0 to the build image? Thanks!

**Comment by rybit** [comment]
The best way to get support for a particular version of ruby is to add a `.ruby_version` to your repo. We will then install (and cache) that one reliably. 

https://github.com/netlify/build-image/blob/master/run-build.sh#L46

**Comment by rybit** [comment]
I misspoke! We only support a few versions. We will revisit this when we look at updating the build script to actually do what I claimed it does.

**Comment by hwhelchel** [comment]
Thank you

**Comment by rybit** [comment]
fixed by #47 
```
$ docker run --rm -it netlify/build:latest /bin/bash
buildbot@643daaf4506f:/$ rvm list

rvm rubies

   ruby-2.0.0-p247 [ x86_64 ]
=* ruby-2.1.2 [ x86_64 ]
   ruby-2.2.1 [ x86_64 ]
   ruby-2.2.3 [ x86_64 ]
   ruby-2.3.0 [ x86_64 ]
   ruby-2.3.1 [ x86_64 ]
   ruby-2.3.3 [ x86_64 ]
   ruby-2.4.0 [ x86_64 ]
```
